### PR TITLE
Show student name in the 'add comment' form #2432

### LINKED ISSUE #2432: Show student name in the 'add comment' form
This helps when the commentor wants to use the student's name in the comment.  Currently, I have to scroll up to see the student's name when I'm commenting on a response that has a few more responses above it.

### PR DESCRIPTION
**Pull request by taniach**
Fixes #2432

### CODE REVIEW

File: src/main/webapp/jsp/instructorStudentRecords.jsp [hunk 1]
```diff
@@ -205,6 +205,7 @@
                                 <div id="commentTextEdit<%=commentIdx%>" style="display:none;">
                                     <div class="form-group form-inline">
                                         <div class="form-group text-muted">
+                                            Comment about <%=InstructorStudentRecordsPageData.sanitizeForHtml(data.student.name)%>:<br>
```
Comment (kai33) [inline src/main/webapp/jsp/instructorStudentRecords.jsp]: inconsistent?
1 br for instructorStudentRecords.jsp, but 2 for instructorFeedbackResults*.jsp

Comment (taniach) [inline src/main/webapp/jsp/instructorStudentRecords.jsp]: 1 br is for comment about a student. 2 br is for comment about a feedback response. Used 2 br when there is a giver and a recipient

Comment (kai33) [inline src/main/webapp/jsp/instructorStudentRecords.jsp]: u used consecutive <br> for instructorFeedbackResults*.jsp, but not for instructorStudentRecords.jsp.

btw it's not a good practice to use <br> here
> Do not use <br> to increase the gap between lines of text; use the CSS margin property or the <p> element.
fyi: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br

Comment (taniach) [inline src/main/webapp/jsp/instructorStudentRecords.jsp]: I will change it to <p> instead

Comment (taniach) [inline src/main/webapp/jsp/instructorStudentRecords.jsp]: P

**Comment by taniach** [comment]
I updated the pull request

**Comment by kai33** [comment]
Hi, can u also update instructorFeedbackResponseCommentsLoad.jsp for this issue?

**Comment by taniach** [comment]
Thanks for pointing it out Kai Xie. I updated it.

**Comment by kai33** [comment]
hi can you squash your commits, merge with the recent master branch and fix the recent InstructorCommentsPageUiTest? 
there's a failed case for testContent() part

**Comment by taniach** [comment]
Ok I updated it

**Comment by kai33** [comment]
hi 1 case failed in instructorFeedbackResultsPageUiTest
at line 133 resultsPage.verifyHtmlMainContent("/instructorFeedbackResultsSortRecipientGiverQuestion.html");

can verify and fix?

**Comment by taniach** [comment]
Fixed it

**Comment by taniach** [comment]
I fixed the test. Would you like me to combine everything into 1 commit?

**Comment by kai33** [comment]
Eg 1~4 commits to show your changes to the UI pages, 1~4 commits to show your changes to the test cases, which is clearer to review

**Comment by kai33** [comment]
Thank you for your contribution :+1: 

fyi: how to squash & rebase commits
http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by taniach** [comment]
Thanks for the link Kai. Sorry I couldn't push the squashed commits because I couldn't resolve the merge conflicts.

**Pull request merged**